modules/ROOT/pages/7.0.1-release-notes.adoc

```diff
@@ -154,12 +154,14 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 [[bug-fixes]]
 == Bug fixes
 
-{productname} {release-version} also includes the following bug fix<es>:
+{productname} {release-version} also includes the following bug fixes:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== The status bar was invisible when the editor's height was set to the minimum.
+// #TINY-10705
 
-// CCFR here.
+In a previous {productname} release, the status bar was moved inside the editor container. Consequently, when the editor's height was set to the minimum and the toolbar contained many buttons, the status bar would be hidden under the toolbar buttons.
```

Comment:
@tiny-ben-tran Was this change specific to `6.3`? if so should we link this back to the release notes when `addView` as added for reference?